### PR TITLE
add program to compare template with fingerprint folfer

### PR DESCRIPTION
**Pull request by itsFDavid**
Added example of program to compare fingerprint with template folder and added more functions how that store image, store template on fingerprint folder, store model on sensor

### CODE REVIEW

File: examples/fingerprint_template_folder_compare_with_file.py [hunk 1]
```diff
@@ -0,0 +1,273 @@
+# SPDX-FileCopyrightText: 2021 ladyada for Adafruit Industries
+# SPDX-License-Identifier: MIT
```
Comment (FoamyGuy) [inline examples/fingerprint_template_folder_compare_with_file.py]:
```suggestion
# SPDX-FileCopyrightText: 2024 itsFDavid
# SPDX-License-Identifier: MIT
```

Comment (FoamyGuy) [reply examples/fingerprint_template_folder_compare_with_file.py]: The copyright should be updated to the correct year and to you

**Review by FoamyGuy — CHANGES_REQUESTED** [review]
Thanks for working on this @itsFDavid. 

It looks mostly good to me, but there are a few tweaks I think should be:

- updating the copyright information
- add "rpi" at the end of the new filename to designate this example is not meant for microcontrollers

File: examples/fingerprint_template_folder_compare_with_file.py (hunk 1 above)
Comment (itsFDavid) [reply examples/fingerprint_template_folder_compare_with_file.py]: ok, thanks for viewed my file, now i change this

**Review by FoamyGuy — APPROVED** [review]
This looks good to me, thank you @itsFDavid!

I didn't test on hardware but seems good to me.

**Pull request merged**